src/main/clm_varctl.F90

```diff
@@ -207,7 +207,7 @@ module clm_varctl
   integer, public            :: fates_parteh_mode = -9                 ! 1 => carbon only
                                                                        ! 2 => C+N+P (not enabled yet)
                                                                        ! no others enabled
-  logical, public            :: use_fates_spitfire = .false.           ! true => use spitfire model
+  integer, public            :: fates_spitfire_mode = 0                ! > 0 => use spitfire model: see bld/namelist_files/namelist_definition_clm4_5.xml for details
```

Comment:
[Same as my most recent comment](https://github.com/ESCOMP/CTSM/pull/991#discussion_r454540753):

Name the various modes in a central location and then use them in all cases like this instead of hardwiring.

Reply:
I completed this in the current PR's latest commit and in https://github.com/NGEET/fates/pull/635.

The model builds, but I get a runtime system error in my cesm.log that looks like a problem leftover from cheyenne's downtime. What do you suggest @ekluzek :
```dplace failed. Verify that the numatools module is loaded.
MPT ERROR: could not run executable (case #1)
        (HPE MPT 2.19  02/23/19 05:31:12)```

Reply:
Good news:
Whatever this was, it's been fixed, and my test now runs :-)